src/views/TradingCompetition/components/Ribbon/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, LaurelLeftIcon, LaurelRightIcon } from '@pancakeswap-libs/uikit'
+import { RibbonProps } from '../../types'
+import {
+  RibbonDownMidExpanding,
+  RibbonUpMidExpanding,
+  RibbonDownRightSide,
+  RibbonUpRightSide,
+  RibbonDownLeftSide,
+  RibbonUpLeftSide,
+} from './RibbonStyles'
+import { Heading2Text, VisuallyHiddenHeading2Text } from '../CompetitionHeadingText'
+
+const Wrapper = styled(Flex)<{ ribbonDirection?: 'up' | 'down' }>`
+  position: relative;
+  display: inline-flex;
+  align-items: ${({ ribbonDirection }) => (ribbonDirection === 'up' ? 'flex-end' : 'flex-start')};
+  justify-content: center;
+
+  svg {
+    z-index: 200;
```

Comment:
Add a comment for why the z-index need to be changed, and why the value 200 is picked

Reply:
I think this was from me debugging some ribbon shiii. Gonna 🗑️ 👍